pkg/identity/cache/allocator.go

```diff
@@ -47,6 +48,17 @@ func (gi globalIdentity) GetKey() string {
 	return kvstore.Encode([]byte(str))
 }
 
+// GetAsMap() encodes a globalIdentity as string
+func (gi globalIdentity) GetAsMap() map[string]string {
+	m := map[string]string{}
+
+	for _, v := range gi.LabelArray {
```

Comment:
Consider making this function as a method of `LabelArray`

Reply:
good idea. I noticed that Labels has a function called StringMap that is exactly this code. I added a StringMap to LabelArray. Not exactly what you had in mind but I didn't want to have two different function names for the same thing. What do you think? (I'll push it up with my other changes soon)